### DISCUSSION TITLE
High DPI support or changing font size?

### DISCUSSION THREAD

**Discussion by yayuwang0624** [icidasset/diffuse#398] [General]
Is high-dpi or font size configurable now？

**Comment by icidasset** [comment]
Not really I guess. Not sure what you mean with high DPI though in relation to an app. I just tend to use the zoom functionality in the browser to increase/decrease the font-size. At some point I want to make this app have themes so things like this can be customised, but that's a huge undertaking I don't have the time for (yet).

**Reply by yayuwang0624** [reply]
yeah, customizable themes will be good but it takes time

Personally I think zoom/scale will be enough for me
